src/content/docs/pt-br/reference/configuration-reference.mdx

```diff
@@ -910,6 +977,100 @@ Veja a documentação completa do objeto de configuração `vite` em [vitejs.dev
 }
 ```
 
+## i18n
+
+<p>
+
+**Tipo:** `object`<br />
+<Since v="3.5.0" />
+</p>
+
+Configura o roteamento i18n e lhe permite especificar algumas opções de customização.
+
+Veja nosso guia para mais informações sobre [internacionalização no Astro](/pt-br/guides/internationalization/)
+
+### i18n.defaultLocale
+
+<p>
+
+**Tipo:** `string`<br />
+<Since v="3.5.0" />
+</p>
+
+A língua padrão do seu site/aplicação. Esse é um campo obrigatório.
```

Comment:
I personally prefer `O idioma` instead of `A língua`. But it's just a suggestion, not very relevant.

Reply:
I kept the terms that were used on other places:
- `locale` -> `língua`
- `language` -> `idioma`